src/components/Dropdown/dropdown.test.tsx

```diff
@@ -79,6 +62,16 @@ describe('<Dropdown />', () => {
     });
   });
 
+  describe('when fully rendered', () => {
+    beforeEach(() => {
+      fullComponent = mount(<Dropdown options={[]} />);
+    });
+
+    it('displays our custom caret icon', () => {
+      expect(fullComponent.find('.y-dropdown__caretDown').length).toBe(1);
```

Comment:
It's not really testing for the icon, but a class. Maybe the desc could be more accurate.

Reply:
this is the class we pass to the YamUI Icon component Dropdown renders, so by selecting this class we're selecting the Icon component's outer element. is there a different wording you'd prefer?